src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Categories/Blocks/form.html.twig

```diff
@@ -105,44 +132,72 @@
       <div class="form-group row">
-        {{ ps.label_with_help(('Meta title'|trans({}, 'Admin.Global')), ('Forbidden characters:'|trans({}, 'Admin.Notifications.Info')) ~ ' <>;=#{}') }}
+        <label class="form-control-label">
+          {{ 'Meta title'|trans({}, 'Admin.Global') }}
+        </label>
         <div class="col-sm">
           {{ form_widget(categoryForm.meta_title) }}
+          <small class="form-text">{{ 'Forbidden characters:'|trans({}, 'Admin.Notifications.Info') ~ ' <>;=#{}' }}</small>
         </div>
       </div>
 
       <div class="form-group row">
-        {{ ps.label_with_help(('Meta description'|trans({}, 'Admin.Global')), ('Forbidden characters:'|trans({}, 'Admin.Notifications.Info')) ~ ' <>;=#{}') }}
+        <label class="form-control-label">
+          {{ 'Meta description'|trans({}, 'Admin.Global') }}
+        </label>
         <div class="col-sm">
           {{ form_widget(categoryForm.meta_description) }}
+          <small class="form-text">
+            {{ 'Forbidden characters:'|trans({}, 'Admin.Notifications.Info') ~ ' <>;=#{}' }}
+          </small>
         </div>
       </div>
 
       <div class="form-group row">
-        {{ ps.label_with_help(('Meta keywords'|trans({}, 'Admin.Global')), ('To add tags, click in the field, write something, and then press the "Enter" key.'|trans({}, 'Admin.Shopparameters.Help')) ~ ('Forbidden characters:'|trans({}, 'Admin.Notifications.Info')) ~ ' <>;=#{}') }}
+        <label class="form-control-label">
+          {{ 'Meta keywords'|trans({}, 'Admin.Global') }}
+        </label>
         <div class="col-sm">
           {{ form_widget(categoryForm.meta_keyword) }}
+          <small class="form-text">
+            {{ 'To add tags, click in the field, write something, and then press the "Enter" key.'|trans({}, 'Admin.Shopparameters.Help') ~ ('Forbidden characters:'|trans({}, 'Admin.Notifications.Info')) ~ ' <>;=#{}' }}
+          </small>
         </div>
       </div>
 
       <div class="form-group row">
-        {{ ps.label_with_help(('Friendly URL'|trans({}, 'Admin.Global')), ('Only letters, numbers, underscore (_) and the minus (-) character are allowed.'|trans({}, 'Admin.Catalog.Help'))) }}
+        <label class="form-control-label">
+          <span class="text-danger">*</span>
+          {{ 'Friendly URL'|trans({}, 'Admin.Global') }}
+        </label>
         <div class="col-sm">
           {{ form_widget(categoryForm.link_rewrite) }}
+          <small class="form-text">
+            {{ 'Only letters, numbers, underscore (_) and the minus (-) character are allowed.'|trans({}, 'Admin.Catalog.Help') }}
+          </small>
         </div>
       </div>
 
       <div class="form-group row">
-        {{ ps.label_with_help(('Group access'|trans), ('Mark all of the customer groups which you would like to have access to this category.'|trans({}, 'Admin.Catalog.Help'))) }}
+        <label class="form-control-label">
+          {{ 'Group access'|trans({}, 'Admin.Global') }}
```

Comment:
Is it possible to localize it in _Admin.Catalog.Feature_ instead?

Reply:
done.